src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/encode/TsvEncoder.scala

```diff
@@ -0,0 +1,214 @@
+        encode(event.tr_country) + "\t" +
+        encode(event.ti_orderid) + "\t" +
+        encode(event.ti_sku) + "\t" +
+        encode(event.ti_name) + "\t" +
+        encode(event.ti_category) + "\t" +
+        encode(event.ti_price) + "\t" +
+        encode(event.ti_quantity) + "\t" +
+        encode(event.pp_xoffset_min) + "\t" +
+        encode(event.pp_xoffset_max) + "\t" +
+        encode(event.pp_yoffset_min) + "\t" +
+        encode(event.pp_yoffset_max) + "\t" +
+        encode(event.useragent) + "\t" +
+        encode(event.br_name) + "\t" +
+        encode(event.br_family) + "\t" +
+        encode(event.br_version) + "\t" +
+        encode(event.br_type) + "\t" +
+        encode(event.br_renderengine) + "\t" +
+        encode(event.br_lang) + "\t" +
+        encode(event.br_features_pdf) + "\t" +
+        encode(event.br_features_flash) + "\t" +
+        encode(event.br_features_java) + "\t" +
+        encode(event.br_features_director) + "\t" +
+        encode(event.br_features_quicktime) + "\t" +
+        encode(event.br_features_realplayer) + "\t" +
+        encode(event.br_features_windowsmedia) + "\t" +
+        encode(event.br_features_gears) + "\t" +
+        encode(event.br_features_silverlight) + "\t" +
+        encode(event.br_cookies) + "\t" +
+        encode(event.br_colordepth) + "\t" +
+        encode(event.br_viewwidth) + "\t" +
+        encode(event.br_viewheight) + "\t" +
+        encode(event.os_name) + "\t" +
+        encode(event.os_family) + "\t" +
+        encode(event.os_manufacturer) + "\t" +
+        encode(event.os_timezone) + "\t" +
+        encode(event.dvce_type) + "\t" +
+        encode(event.dvce_ismobile) + "\t" +
+        encode(event.dvce_screenwidth) + "\t" +
+        encode(event.dvce_screenheight) + "\t" +
+        encode(event.doc_charset) + "\t" +
+        encode(event.doc_width) + "\t" +
+        encode(event.doc_height) + "\t" +
+        encode(event.tr_currency) + "\t" +
+        encode(event.tr_total_base) + "\t" +
+        encode(event.tr_tax_base) + "\t" +
+        encode(event.tr_shipping_base) + "\t" +
+        encode(event.ti_currency) + "\t" +
+        encode(event.ti_price_base) + "\t" +
+        encode(event.base_currency) + "\t" +
+        encode(event.geo_timezone) + "\t" +
+        encode(event.mkt_clickid) + "\t" +
+        encode(event.mkt_network) + "\t" +
+        encode(event.etl_tags) + "\t" +
+        encode(event.dvce_sent_tstamp) + "\t" +
+        encode(event.refr_domain_userid) + "\t" +
+        encode(event.refr_dvce_tstamp) + "\t" +
+        encode(event.derived_contexts) + "\t" +
+        encode(event.domain_sessionid) + "\t" +
+        encode(event.derived_tstamp) + "\t" +
+        encode(event.event_vendor) + "\t" +
+        encode(event.event_name) + "\t" +
+        encode(event.event_format) + "\t" +
+        encode(event.event_version) + "\t" +
+        encode(event.event_fingerprint) + "\t" +
+        encode(event.true_tstamp)
+    ).mkString("\t")
```

Comment:
What am I missing? We're adding a `\t` after every column in `encode(event.foo) = "\t"` and then adding `\t` in `mkString("\t")`? Wouldn't it double amount of tabs?

Reply:
Indeed this `mkString` is a left over from previous implementation. Given that it's applied on a string of length 1 it does nothing but I'll remove it

Reply:
Right, that's a single-element list, that's what I've missed.